commands/output/output.go

```diff
@@ -78,10 +88,29 @@ func getStatusString(r *review.Summary) string {
 }
 
 // PrintSummary prints a single-line summary of a review.
-func PrintSummary(r *review.Summary) {
+func PrintSummary(r *review.Summary, repo *repository.Repo) {
```

Comment:
The `review.Summary` type holds a reference to the repo, so you don't have to pass it in here. Instead, you can just reference `r.Repo`.

Conversely, adding in the additional reference to the repo would be a change to an exported UI, which might break people using git-appraise as a library.

One additional aside; the `repository.Repo` type is an interface, so you would not normally pass it around as a pointer (but that will become a moot point after the switch to using `r.Repo`).